rules_python_zip/simplepack.go

```diff
@@ -501,6 +502,14 @@ if need_unzip and isinstance(__loader__, zipimport.zipimporter):
     tempdir_create_pid = os.getpid()
     atexit.register(clean_tempdir_parent_only, tempdir)
     sys.path.insert(0, tempdir)
+    # Handle linux signal terminate by calling exit, so atexit code executes.
```

Comment:
Better documentation please. Explain why this signal stuff is needed... And the signal-handling code you're adding does not cause anything to do with `atexit` to happen, so I don't think this is correct...